cotainr/tests/pack/stubs.py

```diff
@@ -0,0 +1,41 @@
+"""
+cotainr - a user space Apptainer/Singularity container builder.
+
+Copyright DeiC, deic.dk
+Licensed under the European Union Public License (EUPL) 1.2
+- see the LICENSE file for details.
+
```

Comment:
add a comment on this is a stub for tests.

Reply:
Again, we don't really add much description here. However, the general structure of these files and our general strategy for organizing the tests are described in the documentation:  https://cotainr.readthedocs.io/en/stable/development/test_suite_ci_cd.html

Reply:
same as above. just leave as it is.

Reply:
I'll leave as is